src/app/delivery-history/delivery-history.component.ts

```diff
@@ -315,8 +348,10 @@ export class helperHistoryInfo extends Entity<string>{
   selfassigned = new NumberColumn(getLang(this.context).selfAssigned);
   families = new NumberColumn(getLang(this.context).families);
   dates = new NumberColumn(getLang(this.context).dates);
+  giftsConsumed = new NumberColumn('מתנות שמומשו');
+  giftsPending = new NumberColumn('מתנות זמינות');
   constructor(private context: Context) {
-    super({ name: 'helperHistoryInfo', allowApiRead: false, allowApiCRUD: false });
+    super({ name: 'helperHistoryInfo', allowApiRead: true, allowApiCRUD: false });
```

Comment:
Y?

Reply:
probably not. but not really harmful. anyway - changed it back